### DISCUSSION TITLE
[Feature] Mobile share feature from multi select on album

### DISCUSSION THREAD

**Discussion by Dleewee** [immich-app/immich#1938] [Feature Request]
### The feature

When selecting multiple images in an album on mobile (Android), no share options are presented.

Request to show similar context menu including share when using multi select from album view on mobile. 

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by jon-f-novabackup** [comment]
I think it would be great to have the ability to multi-select photos in Albums (Library) and Sharing views while in thumbnail mode in mobile to share socially, and here is why, not just individually per image one by one which takes forever like if I want to select 10 or 20 images to share socially, meaning share these multi selected 10 or 20 photos that exist in an album to share to one person outside of Immich socially, to like Messenger/WeChat/LINE apps, that is not a user in Immich and I don't want them to be for various reasons, as it was not explained in the original feature request by @Dleewee. First of all the only place in mobile that you can multi select in batch and share assets socially with a user outside of Immich is in the Photos timeline view, but only each individual user can see those assets (you could share them but Sharing and Library views don't have multi-select and share socially capability in mobile as of 1.54.0 build.77 so that is a mute point), so that limits the capability of being able to have anyone be able to multi-select in shared albums and share those images across your entire Immich instance. The shared albums and album (library) views have photos and assets organized in a particular way (grouped), and not everyone has access to those assets, ONLY I can see them and those that are added as members of the shared album can see them, so what if one user that I have added to a shared album wants to share those photos socially to someone not part of the shared album, or what if I want to share photos socially that exist in either my Albums (Library) view or in a Shared album I can see, want to share multiple photos in one go to a social app to someone not even a user in our Immich, it is just a one off here you go I'm sharing you 10 photos from our vacation or day out, and I don't want to add you as a member to my album where you'd need an account and could see everything, I just want to share these 10 photos to you one time, to share to a single person that is not a user in my Immich to share one time socially like that (to like Messenger/WeChat/LINE apps). There is no way to do that in one go in batch unless I view each asset individually and click on the share button and repeat that 10 times in a row; it is also not very seemless to have to do that. **gpshotos, as long as you are a member of a shared album, does allow you to batch / multi-select images / assets in Library and Shared Albums**, to share socially like that and it would be great to have that capability in Immich as well, as to extend that functionality so that it is not just a function that can be done ONLY in each individual Immich user's Photos timeline view (which isn't going to have the photos organized and grouped like they would be in an albums / library type of view, and as I mentioned the other users of my Immich aren't going to be able to share those shared album images either in batch / with multi select because they don't see my Photos timeline view to do that). I also wrote about that need in another discussion thread comment today here: https://github.com/immich-app/immich/discussions/2346#discussioncomment-5756773.

**Comment by bo0tzz** [comment]
This is now supported
